security/cmd/chiron/main.go

```diff
@@ -119,20 +113,19 @@ func init() {
 	flags.DurationVar(&opts.certMinGracePeriod, "cert-min-grace-period",
 		DefaultMinCertGracePeriod, "The minimum certificate rotation grace period.")
 
-	flags.StringVar(&opts.mutatingWebhookServiceNames, "mutating-webhook-service-names", "istio-sidecar-injector",
-		"The names of the services of mutating webhooks, separated by comma. Currently, Chiron will only manage the first one.")
-	flags.StringVar(&opts.validatingWebhookServiceNames, "validating-webhook-service-names", "istio-galley",
-		"The names of the services of validating webhooks, separated by comma. Currently, Chiron will only manage the first one.")
+	flags.StringVar(&opts.serviceNames, "service-names", "istio-galley,istio-sidecar-injector",
+		"The names of the services for which Chiron manage certs.")
+	flags.StringVar(&opts.serviceNamespaces, "serviceNameSpaces", "istio-system,istio-system",
```

Comment:
`StringSliceVar` will release you from parsing it.

Reply:
Done, thanks!